public/js/controllers/mempool.js

```diff
@@ -61,6 +61,47 @@
         $(newRowHtml).insertBefore(rows.first())
     }
 
+    function filterVotesLastBlock(target) {
+        var best_block_hash=$(target).data("hash");
+        var best_block_height=$(target).text();
+        console.log(best_block_height);
+        $('*[data-target="mempool.voteTransactions"] tr').each(function(i,el) {
```

Comment:
If these functions are moved inside this controller class, then you will have the targets available via `this.voteTransactionsTargets`, that would be the stimulus way of accessing the DOM.